assets/locales/en.json

```diff
@@ -141,5 +135,27 @@
   "runGarbageCollectorDone": {
     "title": "Garbage collector",
     "message": "The garbage collector ran successfully."
+  },
+  "downloadCidContentDialog": {
+    "title": "Download",
+    "message": "Type down a CID, IPFS or IPNS path whose contents you want to download.",
+    "action": "Download"
+  },
+  "cantResolveCidDialog": {
+    "title": "Could not resolve",
```

Comment:
I might give the title of all these error dialogs just "Error" to avoid duplication of content.